decoration.sk

```diff
@@ -322,17 +322,23 @@ decoratives after flattening:
 
 village update potted flowers and saplings:
 	minecraft version = 1.14 or newer
-	(potted|flower pot with) bamboo = minecraft:potted_bamboo
-	(potted|flower pot with) cornflower = minecraft:potted_cornflower
-	(potted|flower pot with) lily of the valley = minecraft:potted_lily_of_the_valley
-	(potted|flower pot with) wither rose = minecraft:potted_wither_rose
+	(potted|flower pot with [a]) bamboo = minecraft:potted_bamboo
+	(potted|flower pot with [a]) cornflower = minecraft:potted_cornflower
+	(potted|flower pot with [a]) lily of the valley = minecraft:potted_lily_of_the_valley
+	(potted|flower pot with [a]) wither rose = minecraft:potted_wither_rose
 	(any flower pot|[any] potted plant) = any potted plant, potted bamboo, potted cornflower, potted lily of the valley, potted wither rose
 	bamboo sapling = minecraft:bamboo_sapling
 
 village and pillage decoratives:
 	minecraft version = 1.14 or newer
 	{waterloggable} {directional} {wood type} wall sign¦s = -wall_sign
 	{waterloggable} {rotatable} {wood type} [floor] sign¦s = -sign
+	oak sign¦s = oak floor sign, oak wall sign
+	birch sign¦s = birch floor sign, birch wall sign
+	jungle sign¦s = jungle floor sign, jungle wall sign
+	spruce sign¦s = spruce floor sign, spruce wall sign
+	acacia sign¦s = acacia floor sign, acacia wall sign
+	dark oak sign¦s = dark oak floor sign, dark oak wall sign
```

Comment:
Is this necessary? It should be covered by the floor sign alias since the `floor` part is optional

Reply:
Then [floor] shouldn't be optional

Reply:
omg, it gets set to non optional on the aquatic signs tag anyways, so yup.

Reply:
I mean should be okay as long as it works properly with giving the item to the player and setting the block in the world